websites/rawkode.dev/src/styles/global.css

```
@@ -40,42 +57,58 @@
     text-underline-offset: 3px;
-    transition: text-decoration-color var(--durations-fast) var(--easings-out-expo);
+    transition:
+      color var(--durations-fast) var(--easings-out-expo),
+      text-decoration-color var(--durations-fast) var(--easings-out-expo);
   }
-  a:hover { text-decoration-color: currentColor; }
 
-  strong { font-weight: 600; color: var(--colors-text-primary); }
+  a:hover {
+    color: var(--colors-accent);
+    text-decoration-color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)